libnet/sample/Makefile.am

```diff
@@ -77,3 +77,9 @@ test_ipv6_icmpv4_SOURCES = test_ipv6_icmpv4.c
 EXTRA_DIST = libnet_test.h
 
 LDADD 			= $(top_builddir)/src/libnet.la
+
+EXTRA_DIST += arp-new.c win32/arp/arp.vcproj win32/cdp/cdp.vcproj	\
```

Comment:
Okay, I rebased and updated and squashed and force pushed.
There are now a small handful of doxygen targets when it is enabled, and none when it isn't.
Also the code is smaller and more transparent, I hope.